analyzers/src/SonarAnalyzer.CSharp/Rules/SqlKeywordsDelimitedBySpace.cs

```diff
@@ -120,6 +120,25 @@ public StringConcatenationWalker(SyntaxNodeAnalysisContext context)
                 this.context = context;
             }
 
+            public override void VisitInterpolatedStringExpression(InterpolatedStringExpressionSyntax node)
+            {
+                var strings = new List<StringWrapper>();
+                foreach (var content in node.Contents)
+                {
```

Comment:
this happens only when the `using` with SQL namespace is present, right?

Reply:
Yes, the syntax walker is only triggered if usings with SQL namespaces are present.